src/meta/src/hummock/model/branched_sst_info.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use prost::Message;
+use risingwave_hummock_sdk::HummockSstableId;
+use risingwave_pb::hummock::BranchedSstInfo;
+
+use crate::model::{MetadataModel, MetadataModelResult};
+
+/// `cf(branched_sst_info)`: `HummockSsTableId` -> `BranchedSstInfo`
+const HUMMOCK_BRANCHED_SST_INFO: &str = "cf/branched_sst_info";
+
+impl MetadataModel for BranchedSstInfo {
```

Comment:
MetadataModel is unneccessary for a pure in-mem structure.

Reply:
> MetadataModel is unneccessary for a pure in-mem structure.

fixed